Documentation/Reference/Columns/Check/Index.rst

```diff
@@ -159,6 +159,8 @@ showIfRTE
          If set, this field will show *only* if the RTE editor is enabled
          (which includes correct browser version and user-rights altogether.)
 
+         This property is deprecated since TYPO3 v8.
```

Comment:
@lolli42 maybe we could rethink that concept, because its the same amount of work (removing the text or adding the "deprecated since 8.3") and the latter would help a lot people. Usually finding code snippets on the net you end up looking into the documentation and not finding the "property" documented you might think that its just "undocumented feature". 

I know this is not the place to discuss this, but just as an idea for the future.